virttest/qemu_storage.py

```diff
@@ -457,6 +522,24 @@ def _secret_objects(self):
         secret_obj_str = "--object secret,id={s.aid},data={s.data}"
         return [secret_obj_str.format(s=s) for s in secret_objects]
 
+    @property
+    def _image_access_tls_creds_object(self):
+        """Get the tls-creds object str of the image itself."""
+        tls_obj_str = '--object tls-creds-x509,id={s.aid},endpoint=client,dir={s.tls_creds}'
```

Comment:
Should we support other types of TLS handshake credentials? I saw there are others TLS credentials from upstream qemu codes.
```
tls-creds-anon: Creates a TLS anonymous credentials object, which can be used to provide TLS support on network backends.
tls-creds-psk: Creates a TLS Pre-Shared Keys (PSK) credentials object, which can be used to provide TLS support on network backends.
```
Suggest adding argument `tls_creds_name` to  `_image_access_tls_creds_object` if we wanna support them avoid hard code.

Reply:
we now only support tls-creds-x509.
If we have to support other objects in future, we have to add a new param like nbd_tls_creds_object to differentiate objects, these information will be stored in storage.StorageAuth, we can get it from StorageAuth object, i.e., the 's' object in _image_access_tls_creds_object.

Also, I added a TODO in StorageAuth to comment this.

Reply:
Got it, thanks for your explanations.